apps/src/applab/applab.js

```diff
@@ -1052,6 +1052,7 @@ Applab.resetTurtle = function() {
   Applab.turtle.heading = 0;
   Applab.turtle.x = Applab.appWidth / 2;
   Applab.turtle.y = Applab.appHeight / 2;
+  Applab.turtle.penDown = true; // turtle defaults to penDown on reset
```

Comment:
This seems odd.  Wouldn't we want to wait until a "mousedown" to start drawing?

Reply:
On production, this is the default behavior.

Reply:
Ah, I hadn't realised that app lab had a full turtle implementation.  This makes sense for that.